ClientLib/src/main/java/com/o3dr/services/android/lib/drone/attribute/AttributeEvent.java

```diff
@@ -69,6 +69,7 @@
     public static final String MISSION_SENT = PACKAGE_NAME + ".MISSION_SENT";
     public static final String MISSION_RECEIVED = PACKAGE_NAME + ".MISSION_RECEIVED";
     public static final String MISSION_ITEM_UPDATED = PACKAGE_NAME + ".MISSION_ITEM_UPDATED";
+    public static final String MISSION_ITEM_REACHED = PACKAGE_NAME + ".MISSION_ITEM_UPDATED";
```

Comment:
@squilter the value for the `MISSION_ITEM_REACHED` constant is the same as the one for the `MISSION_ITEM_UPDATED` constant.